src/components/EventRegistrantsModal/EventRegistrantsModal.tsx

```
@@ -43,6 +44,14 @@
   const [member, setMember] = useState<InterfaceUser | null>(null);
+  const [isAdding, setIsAdding] = useState(false);
+  const [addRegistrantMutation] = useMutation(ADD_EVENT_ATTENDEE, {
+    refetchQueries: [
+      { query: EVENT_ATTENDEES, variables: { id: eventId } },
+      { query: MEMBERS_LIST, variables: { id: orgId } },
+    ],
+  });
+  const [open, setOpen] = useState(false);
 
   // Hooks for mutation operations
   const [addRegistrantMutation] = useMutation(ADD_EVENT_ATTENDEE);
```

Biome

[error] Shouldn't redeclare 'addRegistrantMutation'. Consider to delete it or rename it.

'addRegistrantMutation' is defined here:


(lint/suspicious/noRedeclare)